code/modules/hydroponics/seeds.dm

```diff
@@ -209,17 +211,21 @@
 		product_count = clamp(round(product_count/2),0,5)
 	while(t_amount < product_count)
 		var/obj/item/reagent_containers/food/snacks/grown/t_prod
-		if(instability >= 30 && prob(instability/3) && mutatelist)
+		if(instability >= 30 && prob(instability/3) && length(mutatelist) && (seed_flags & MUTATE_EARLY))
```

Comment:
```suggestion
		if(instability >= 30 && seed_flags & MUTATE_EARLY && length(mutatelist) && prob(instability/3))
```
Try to order checks for cost and relevance. No need to run a probability if the event has no potential to happen, for example.